README.rst

```diff
@@ -32,48 +38,34 @@ Reporting Security Issues
 
 Please do not report security issues in public. Please email security@edx.org.
 
-Get Help
---------
-
-Ask questions and discuss this project on `Slack <https://openedx.slack.com/messages/general/>`_ or in the `edx-code Google Group <https://groups.google.com/forum/#!forum/edx-code>`_.
-
 Getting Started
 ---------------
+Journal currently runs in it’s own docker environment. For development, it is dependent on the edx-platform which also runs in it’s own docker environment. Initial setup is necessary to configure both environments such that they can communicate with each other.
+
+edx-platform configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+1. Get edx-platform docker devstack up and running.
+   
+   -  ``git clone https://github.com/edx/devstack.git; git checkout whitelabel/journals``
+   -  Then follow instructions found here to configure: https://github.com/edx/devstack/blob/master/README.rst
+
+2. Verify that edx-platform is working correctly. Visit http://localhost:18000 and make sure you can login
+
+3. Install whitelabel/journal branches for the following three services in the directories where devstack is configured to look:
+
+   -  LMS: ``cd ~/projects/edx-platform; git checkout whitelabel/journals``
+   -  Discovery: ``cd ~/projects/course-discovery; git checkout whitelabel/journals``
+   -  Ecommerce: ``cd ~/projects/ecommerce; git checkout whitelabel/journals``
+
+4. Restart the services: ``docker-compose restart``
+
+Journal Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+1. Run ``make dev.destroy`` to destroy previous setup and start from scratch
+
+2. Run ``make dev.provision`` to setup all necessary configuration in Journals and edx-platform and create ‘Demo Journal’ 
+
+3. Verify Journal is running by going to http://localhost:18606. You should see “edx Index” page by default showing one Journal “Demo Journal”
+
+4. To edit the Demo Journal, click the CMS link in the header and login as staff@example.com/edx
```

Comment:
CMS link only shows if you are logged in as a staff user. So switch the login step and the link clicking.